### PR TITLE
LPS-188260 Delete Aggregation Object Field when deleting Object Relationship

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPS-188260
Previous work: https://github.com/liferay-objects/liferay-portal/pull/2379

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @carolmariaabb, do you mind in taking a look at this work? Do you see improvements? 
FYI: Product team decided that: when deleting Object Relationship, then delete all the aggregation fields related to that relationship. Slack thread: https://liferay.slack.com/archives/C026UR5976G/p1689605765699809

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -314,6 +318,42 @@ public void deleteObjectFieldByObjectDefinitionId(Long objectDefinitionId)
 		}
 	}
 
+	@Override
+	public void deleteObjectFieldByObjectRelationshipId(
+			long objectRelationshipId)
+		throws PortalException {
+
+		List<Long> objectFieldIds = objectFieldPersistence.dslQuery(
```
[the remark this answers is not in the document]
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:326]: Thank you for the suggestion :smile:

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
@marcelabc @carolmariaabb @G-Sa  I remember the designers defined that we should display a popup to alert that other fields will be deleted. Are we doing it?

**Comment by marcelabc** [comment]
I explained to @gabrielwas that there is a pop-up explaining that "related fields are going to be deleted" :+1: 
However, this pop-up does not show which fields will be impacted. We can improve this later if needed it.

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**